pkg/option/config.go

```diff
@@ -245,6 +245,12 @@ const (
 	// MTUName is the name of the MTU option
 	MTUName = "mtu"
 
+	// DatapathMode is the name of the DatapathMode option
+	DatapathMode = "datapath-mode"
+
+	// DatapathMode is the name of the DatapathMode option
```

Comment:
Small comment change left but we can do that after the CI passes if this is the only change left.